build-support/reapi-sample-server/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.9.12-slim-buster@sha256:62ed2b347a385102d33f5e82530862359f8dc60464674d78cef844b02d150a50
```

Comment:
in my branch, I created `build-support/docker/`, which we used to do a while ago: https://github.com/pantsbuild/pants/tree/toolchain-remote-execution

Reply:
I don't find "docker" to be specific enough in this case, since it's more of an implementation detail.